go/flexibletable/table.go

```diff
@@ -42,6 +43,71 @@ func (t *Table) Insert(row Row) error {
 	return nil
 }
 
+func (t *Table) breakOnLineBreaks() error {
+
+	// so that there's no need to resize if there's no line break
+	broken := make([]Row, 0, len(t.rows))
+
+	for _, row := range t.rows {
+
+		for notEmpty := true; notEmpty; {
```

Comment:
Sounds good; I'll change the `for wrapping := true; wrapping; {` in the same file too then